example7/spdx/example7-bin.spdx.json

```
@@ -41,19 +41,19 @@
   ],
   "packages": [
     {
-      "packageName": "hello",
+      "name": "hello",
       "SPDXID": "SPDXRef-go-bin-hello",
-      "downloadLocation": "git@github.com:swinslow/spdx-examples.git#example7/content/build/hello",
+      "downloadLocation": "git://git@github.com:swinslow/spdx-examples.git#example7/content/build/hello",
       "filesAnalyzed": false,
-      "packageLicenseConcluded": "NOASSERTION",
-      "packageLicenseDeclared": "NOASSERTION",
-      "packageCopyrightText": "NOASSERTION"
+      "licenseConcluded": "NOASSERTION",
+      "licenseDeclared": "NOASSERTION",
+      "copyrightText": "NOASSERTION"
     }
   ],
   "relationships": [
     {
-      "spdxElementId": "DocumentRef-golang-dist",
-      "relatedSpdxElement": "DocumentRef-hello-go-module",
+      "spdxElementId": "DocumentRef-golang-dist:SPDXRef-DOCUMENT",
+      "relatedSpdxElement": "DocumentRef-hello-go-module:SPDXRef-DOCUMENT",
```

As noted these should point to `DocumentRef:PackageRef`:

```suggestion
      "spdxElementId": "DocumentRef-golang-dist:SPDXRef-golang-dist",
      "relatedSpdxElement": "DocumentRef-hello-go-module:SPDXRef-go-module-example.com-hello",
```